src/etc/inc/ipsec.inc

```diff
@@ -196,6 +196,7 @@ $p1_authentication_methods = array(
 	'eap-radius' => array('name' => gettext('EAP-RADIUS'), 'mobile' => true),
 	'eap-mschapv2' => array('name' => gettext('EAP-MSChapv2'), 'mobile' => true),
 	'cert' => array('name' => gettext('Mutual Certificate'), 'mobile' => false),
+	'pkcs11' => array('name' => gettext('Mutual Certificate (Token)'), 'mobile' => false),
```

Comment:
This should probably say `PKCS#11 Token` here, rather than only `Token` which may be unclear/confusing.

Reply:
done